Resources/public/js/ezrecommendation.js

```diff
@@ -0,0 +1,77 @@
+// register HandleBars helpers when library is loaded
+
+document.addEventListener('DOMContentLoaded', function() {
+    Handlebars.registerHelper('each', function(context, options) {
+        var result = '';
+        for(var i = 0, j = context.length; i < j; i++) {
+            result = result + options.fn(context[i]);
+        }
+        return result;
+    });
+
+    Handlebars.registerHelper('formatDate', function(timestamp) {
+        var fix = (timestamp + '000') * 1;
```

Comment:
UNIX timestamps are expressed in seconds, but JavaScript works in milliseconds.
I must multiply by 1 to get integer instead of string.